eng/performance/maui_scenarios_android.proj

```diff
@@ -25,19 +29,19 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <MAUIAndroidScenario Include="Maui Android Default">
-      <ScenarioDirectoryName>mauiandroid</ScenarioDirectoryName>
-      <PayloadDirectory>$(ScenariosDir)%(ScenarioDirectoryName)</PayloadDirectory>
-      <ApkName>com.companyname.mauiandroiddefault-Signed</ApkName>
-      <PackageName>com.companyname.mauiandroiddefault</PackageName>
-    </MAUIAndroidScenario>
-    <MAUIAndroidScenario Include=".NET Android Default">
+    <MAUIAndroidScenario Include=".NET Android Default $(RunConfigsString)">
       <ScenarioDirectoryName>netandroid</ScenarioDirectoryName>
       <PayloadDirectory>$(ScenariosDir)%(ScenarioDirectoryName)</PayloadDirectory>
       <ApkName>com.companyname.netandroiddefault-Signed</ApkName>
       <PackageName>com.companyname.NetAndroidDefault</PackageName>
     </MAUIAndroidScenario>
-    <MAUIAndroidScenario Include="Maui Blazor Android Default">
+    <MAUIAndroidScenario Include="Maui Android Default $(RunConfigsString)" Condition="'$(RuntimeFlavor)' == 'mono'">
+      <ScenarioDirectoryName>mauiandroid</ScenarioDirectoryName>
+      <PayloadDirectory>$(ScenariosDir)%(ScenarioDirectoryName)</PayloadDirectory>
+      <ApkName>com.companyname.mauiandroiddefault-Signed</ApkName>
+      <PackageName>com.companyname.mauiandroiddefault</PackageName>
+    </MAUIAndroidScenario>
+    <MAUIAndroidScenario Include="Maui Blazor Android Default (TTFD) $(RunConfigsString)" Condition="'$(RuntimeFlavor)' == 'mono'">
```

Comment:
Why do we only care about TTFD measurement here? Or why don't we measure both TTID and TTFD?

Reply:
I don't know. I just saw that it is passing the `--use-fully-drawn-time` so I added it to the scenario name to make it clear. I can probably add TTID as well if this is something we want.

Reply:
@LoopedBard3 do you remember why for blazor we are using `--use-fully-drawn-time` and not the default startup measurement config?

Reply:
I don't recall anything specifically. Maybe TTID doesn't include some of the blazor startup. Regardless, I don't see any reason not to include both.